test/service/test_manifest.py

```diff
@@ -351,6 +351,13 @@ def test_manifest_zarr(self):
                     self.assertEqual(11, len({row['file_sha256'] for row in rows}), 'Incorrect unique sha256 count')
                     self.assertEqual(11, len({row['file_crc32c'] for row in rows}), 'Incorrect unique crc32c count')
 
+                    response = self._get_manifest(ManifestFormat.curl, filters)
+                    self.assertEqual(200, response.status_code, 'Unable to download manifest')
```

Comment:
```suggestion
                    self.assertEqual(200, response.status_code)
```

Here and above.

This is fluff. If someone needs that message to diagnose the assertion failure, they are working in the wrong profession.

Reply:
In fact, remove all of these messages in this test.

Reply:
This file is full of lines like 
```
self.assertEqual(200, response.status_code, 'Unable to download manifest')
```

Fixing them all might be beyond the scope of this PR, but consider adding another commit that removed the "fluff" from all such lines.